capabilities_matrix/_topics/cloud_providers.md

```diff
@@ -2,60 +2,60 @@
+| Provision from Image to Instance           | Yes                       | Yes             | Yes                         | Yes                        | Yes                                   | Yes                               |
+| Provision from Image Snapshot to Instance  | No                        | No              | No                          | Yes                        | No                                    | No                                |
+| Provision from Volume to Instance          | No                        | No              | No                          | Yes                        | No                                    | No                                |
+| Provision from Volume Snapshot to Instance | No                        | No              | No                          | Yes                        | No                                    | No                                |
+| Orchestration Template Provisioning        | Yes                       | Yes             | No                          | Yes                        | No                                    | No                                |
+| Cloud-Init                                 | No                        | No              | No                          | Yes                        | No                                    | Yes                               |
+| Sysprep Windows Templates                  | No                        | No              | No                          | Yes                        | No                                    | No                                |
+| Network Manager Relationships              | Yes                       | Yes             | Yes                         | Yes                        | No                                    | Yes                               |
+| Relationship Discovery                     | Yes                       | Yes             | Yes                         | Yes                        | No                                    | Yes                               |
+| Cloud Network Inventory                    | Yes                       | Yes             | Yes                         | Yes                        | No                                    | Yes                               |
+| Cloud Subnet Inventory                     | Yes                       | Yes             | Yes                         | Yes                        | No                                    | Yes                               |
+| Network Router Inventory                   | No                        | No              | Yes                         | Yes                        | No                                    | No                                |
+| Security Groups                            | Yes                       | Yes             | Yes                         | Yes                        | No                                    | No                                |
```

Comment:
Should we say N/A as this isn't supported by PowerVS - no somewhat gives the implication that it's a gap?

Reply:
(Security Groups)

Reply:
done